src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/elasticapi/ElasticExtensions.kt

```diff
@@ -107,6 +48,18 @@ fun IndexMetadata.getManagedIndexMetaData(): ManagedIndexMetaData? {
     return null
 }
 
+fun getClosedIndices(state: ClusterState): MutableList<String> {
```

Comment:
Change to `getClosedIndicesUuids` (or is it getClosedIndexUuids 🤔 @qreshi help)

Reply:
`getUuidsForClosedIndices` maybe?

Reply:
changed to getUuidsForClosedIndices